system/doc/reference_manual/expressions.xml

```diff
@@ -1390,14 +1390,15 @@ Ei = Value |
       <item>For <c>binary</c> and <c>bitstring</c> it is
       the whole binary or bit string.</item>
     </list>
-    <p>In matching, this default value is only
-    valid for the last element. All other bit string or binary
-    elements in the matching must have a size specification.</p>
+    <p>In matching, the default value for a a binary or bit string
+    segment is only valid for the last element. All other bit string
+    or binary elements in the matching must have a size
+    specification.</p>
```

Comment:
(This is a suggestion, take it or leave it)
I think this is understandable but, for newcomers, I would add here an example that shows what you mean. That is, an example where you show other bit string / binary elements in the matching with their size specification and also the last element with the specification.

Reply:
I will add an example.